src/js/review/ReviewChapters.jsx

```diff
@@ -125,8 +125,8 @@ function mapStateToProps(state, ownProps) {
   const chapters = chapterNames.map(chapterName => {
     const pages = pagesByChapter[chapterName];
 
-    const activePages = getActivePages(pages, formData);
-    const expandedPages = expandArrayPages(activePages, formData);
+    let expandedPages = expandArrayPages(pages, formData);
```

Comment:
Why `let` here and not `const`? You can always nest the `expandArrayPages()` in the next line.

Reply:
Nesting seemed less readable than this, but I'm happy to change it if you'd prefer.